lib/dotcom_web/components/system_status/commuter_rail_timetable_status.ex

```diff
@@ -0,0 +1,159 @@
+defmodule DotcomWeb.Components.SystemStatus.CommuterRailTimetableStatus do
+  @moduledoc """
+  Component displaying the status of a specific commuter rail line.
+  """
+
+  use DotcomWeb, :component
+
+  import DotcomWeb.Components, only: [bordered_container: 1, unstyled_accordion: 1]
+  import DotcomWeb.Components.Alerts, only: [embedded_alert: 1]
+  import DotcomWeb.Components.SystemStatus.StatusLabel, only: [status_label: 1]
+
+  alias Alerts.Alert
+
+  attr :status, :map
+
+  def commuter_rail_timetable_status(assigns) do
```

Comment:
I don't really love this name, but `commuter_rail_line_status` and `commuter_rail_route_status` felt weird in the minute-of.

Actually now that I think about it, `commuter_rail_route_status` makes sense to me - I don't remember why I didn't like it at first.